hologram-minecraft-server.pkg.toml

```diff
@@ -20,9 +20,154 @@ group = "users"
+    command="/home/minecraft/bin/minecraftServerLauncher.sh",no-port-forwarding,no-X11-forwarding,no-agent-forwarding,no-pty ssh-rsa AAAAB3NzaC1yc2EAAAABIwAAAgEAtD1Wf7vCrS8iQwDS3D+qt8WS2hOdLGUTz8XkjX4vDs5wBvcwvvUXnChSMJNYLQFuSw49XlasfGaDeZsPX+CUH/hozQF4y9TIy+/RinaA0rWFiEok6xB3trmcTDYMi43PxZqBqy5HIK0mhKEbCRcp9FNB5qhLKxOphARfXFtGPo2BulLpZpnTMFE6jJrWylcXOkb3C4+A/e71ztO8hwYemPWoYl0lqJMtPS1KoifjIpEWW5JkcvmS2MbXBAfKf/97YtXDBiLh4WRA8BsKM8/V0ZxNOpeCwxVs9lQMCkBzl7374A4OjAEbh7ie3BDr9IEYH4ofH+aAaO05IUsbCZvGn55dhCt+T1L7a9jDd6RsUJwazOHalqCXqN/4imEzhSFYE6B5QdbhqHL6+mhKG3SenjSWv49hvEojU/7XohLPvBgnvupTNktjDTBG+fUfuL2XbJRZ91d6tYZkjmlmNJox6dNE4WzHLd73WywUW3s7OxWIaz8vBvnIlRxXVSbG/Irxq7YPx7+L/rFsj55cchD2FEbgtBfbNiG88/49WyK3M+5yzX7fTdNPVMeNjEYA1A6+eqU0LauOejTA9Zp7ernBjl40i/lXSectBipZFUL7azRSE7RzY53Y7CeLS68HrmVos3mSO0rMU/Fk11xHodImgH/KbjNiIeArAq6uYRi7eRk= apophis@schlepptop
+    command="/home/minecraft/bin/minecraftServerLauncher.sh",no-port-forwarding,no-X11-forwarding,no-agent-forwarding,no-pty ssh-ed25519 AAAAC3NzaC1lZDI1NTE5AAAAIBuoAQ9uQayjVuvFwnfV+pXrauYIvge6DGk9gdapIhoO apophis@toruk
+    command="/home/minecraft/bin/minecraftServerLauncher.sh",no-port-forwarding,no-X11-forwarding,no-agent-forwarding,no-pty ssh-rsa AAAAB3NzaC1yc2EAAAADAQABAAABAQCs0PiLBRaS63NBDLYhXDzIvKT2a6rHbiosOiKIjjiOpn5LLZZPgaOfghp3z38ZwiUC5b3peL86TFC905zYU7KXXAoOGRVAT7dn+emCPcGn6164K2nDPGA1GvZP7gS2gUhvrxmDw66ilt32WWTyf4Xv5xwyOO8S0a98jc+MAib9ksETciea3XSAVswKUCG3SAR1KIru7qNPaPuGxu6QWQEjZgjkHTZ0IL7v82cNm/XxtTKtJDa4LRegxNUJBYOuz36C8kx6mlM3Y4K+d5eevOJKKjlldr2NWj+3GJbpV2ItO6fed/MzMCJLep61l5Fvr4RHWFZhHaPJ7QkUrqBnX4Kn felix
+'''
+
+[[file]]
+path = "/home/minecraft/bin/minecraftServerLauncher.sh"
+content = '''
+#!/bin/bash
+
+# CONFDIR="$HOME/.minecraftServerLauncher/"
+# if [ ! -d "$CONFDIR" ]; then
+# 	mkdir "$CONFDIR"
+# fi
+
+CONFFILE="$HOME/.minecraftServerLauncherrc"
+if [ -f "$CONFFILE" ]; then
+	. "$CONFFILE"
+else
+	SERVERROOT="$HOME/minecraft"
+fi
+
+STATEFILE="$XDG_RUNTIME_DIR/minecraftServerState"
+LOCKFILE="$XDG_RUNTIME_DIR/minecraftServerLock"
+
+# always call this when exiting after a lock was aquired to ensure it is
+# cleaned-up properly
+exitHandler()
+{
+	rm "$LOCKFILE"
+	flock -u 9
+	# echo EXIT
+	if [ -z "$1" ]; then
+		exit 0
+	else
+		exit $1 || exit 1
+	fi
+}
+
+showAvail()
+{
+	echo "Available Servers:"
+	ls "$SERVERROOT"
+}
+
+lock()
+{
+	exec 9> "$LOCKFILE"
+	if ! flock -n 9 ; then
+		echo "Locked by other process."
+		exit 1
+	fi
+}
+
+getState()
+{
+	if [ ! -f "$STATEFILE" ]; then
+		CURRENTSERVER=""
+	else
+		CURRENTSERVER=$( cat "$STATEFILE" )
+		set $CURRENTSERVER
+		CS_NAME=$1
+		CS_PID=$2
+	fi
+}
+
+trap exitHandler SIGINT
+trap exitHandler SIGTERM
```

Comment:
You want `trap exitHandler EXIT`. `EXIT` is a special sigspec recognized by the trap builtin which makes the exitHandler run when the shell exits, for whatever reason.